src/amp/sagas/share.js

```diff
@@ -1,5 +1,13 @@
 /* eslint-disable no-underscore-dangle */
```

Comment:
You can delete this file, right?

Reply:
You are completely right. I'll fix this.

Reply:
True.